### DISCUSSION TITLE
api filter: "_between" does not seems to be inclusive

### DISCUSSION THREAD

**Discussion by SavArmadillo** [directus/directus#13637] [Archive]
Hello!

I am using directus `9.11.1` with a postgresql database.

After reading the documentation on the api filter rules, I encountered a problem while calling the api using the `_between` operator on dateTime values.

Using postman, I created this simple query using the `SEARCH` option to try it out:
```json
{
    "query": {
        "filter": {"time": {"_between": ["2022-05-25", "2022-05-31"]}},
    "fields": ["time"],
    "sort": ["-time"]
    }
}
```
The result I got where values between 2022-05-25 and 2022-05-31 but with the latter excluded:
```json
{
    "data": [
        {
            "time": "2022-05-30T18:08:16"
        },
        ...
        {
            "time": "2022-05-25T06:45:55"
        }
     ]
}
```

I can confirm that I have values in the 2022-05-31 date.

Is there a problem with my query or should I look somewhere else?

**Comment by rijkvanzanten** [comment]
I'm pretty sure the time will default to `00:00:00` if not specified, so `2022-05-31T18:08:16` won't be between `2022-05-25` and `2022-05-31`, as it's past `00:00`

**Reply by SavArmadillo** [reply]
oh okay i'll take that to account then, thanks!
